haf/bus.py

```diff
@@ -57,7 +57,7 @@ def start_manager_server(self):
         # runner queue to web server
         publish_runner = Queue(maxsize=1)
         # case count queue
-        case_count = Queue(maxsize=1)
+        case_count = Queue(maxsize=-1)

```

Comment:
- here is the case_count queue to block the recorder!
- so make the size of queue to the max-size